spec/views/v1/collections/_collection.json.jbuilder_spec.rb

```diff
@@ -5,12 +5,11 @@
     render partial: 'v1/collections/collection', locals: { collection: Purl.first }
     expect(JSON.parse(rendered)).to include(
       'druid' => 'druid:ee111ff2222',
-      'true_targets' => ['SearchWorksPreview'],
       'catkey' => ''
     )
   end
-  it 'always returns "SearchWorksPreview"' do
-    render partial: 'v1/purls/purl', locals: { purl: Purl.first }
+  it 'always returns "SearchWorksPreview" for non deleted Purls' do
+    render partial: 'v1/collections/collection', locals: { collection: Purl.find(6) }
```

Comment:
Ah.. ok, I can do that. The reason is that we really just want a `Purl` that isn't deleted. The other. Our "collections" are really just `Purl` with an association. And modifying the collection (to undelete) broke more tests.. so I opted for this easier path.